pkg/kudoctl/packages/verifier/template/verify_parameters.go

```diff
@@ -81,7 +81,10 @@ func paramsDefinedNotUsed(pf *packages.Files) verifier.Result {
 	}
 	for _, value := range pf.Params.Parameters {
 		if _, ok := tparams[value.Name]; !ok {
-			res.AddParamWarning(value.Name, "defined but not used.")
+			// A parameter could be use to trigger a plan while not being used in templates.
+			if value.Trigger == "" {
```

Comment:
shouldn't we warn if the plan does not exist?

Reply:
We do that already, but in a different validation check that returns an error if a trigger is set to a non-existing plan.